.github/workflows/gh-ci.yaml

```diff
@@ -211,9 +212,57 @@ jobs:
 
     - name: deploy_docs
       if: github.event_name != 'pull_request'
+      env:
```

Comment:
Moved deployment to within this file because it saved a lot of variable checking and possibly insecure token passing